styles.css

```diff
@@ -88,17 +117,603 @@ h4,
 }
 
 .social {
-  height: 1.5em;
-  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  width: 40px;
+  height: 40px;
+  border-radius: 8px;
 }
 
 .sec-container {
-  background-image: url("./img/illustration/Illustration.svg");
+  background-image: url("./img/illustration/Group.svg");
   background-repeat: no-repeat;
   background-position-x: right;
   background-position-y: bottom;
-  background-size: 50% 50%;
+  background-size: 45% 45%;
+  width: 100%;
+  background-color: #1c1a19;
+}
+
+/* Styles for work section. For mobile viewport only */
+.main-container {
+  display: grid;
+  grid-template-rows: auto auto auto auto auto auto auto;
+  width: 100%;
+  height: 3455px;
+}
+
+.tittle {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 10px;
+  gap: 15px;
+  width: 327px;
+  height: 107px;
+  margin: 35% auto auto;
+}
+
+.tittle-header {
+  width: 327px;
+  height: 83px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+}
+
+.victor {
+  width: 327px;
+  height: 0;
+  border: 1px solid #212121;
+}
+
+.work-cards {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 80px;
+  gap: 24px;
+  width: 200px;
+  height: 3028px;
+  margin: 0 auto;
+}
+
+.card1-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0 24px;
+  gap: 24px;
   width: 100%;
+  height: 568px;
+  margin: 0 auto;
+}
+
+.works-img {
+  width: 327px;
+  height: 237px;
+}
+
+.card1-content {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  width: 327px;
+  height: 107px;
+  margin: 0 auto;
+}
+
+.works-header {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+}
+
+.works-header,
+.works-txt,
+.tech {
+  margin-bottom: 5%;
+}
+
+.works-txt {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  width: 327px;
+  height: 147px;
+  color: #091e42;
+}
+
+.tech {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 297px;
+  height: 40px;
+}
```

Comment:
Please consider making the color of all the list items in accordance with the Figma Template design you have choosed to make it look alike to the template.

## Solution:--

You can just add the following piece of CSS in the above `.tech class`

```
    color: #091E42;
```

## Your Design:--

![image](https://user-images.githubusercontent.com/94223367/220148593-f2a593b9-f5b5-4c54-b176-0e4396427175.png)